outwatch/src/main/scala/outwatch/AsVDomModifier.scala

```diff
@@ -26,14 +25,14 @@ object AsVDomModifier {
   }
 
   implicit def arrayModifier[T : AsVDomModifier]: AsVDomModifier[Array[T]] = (value: Array[T]) =>
-    CompositeModifier(value.map(VDomModifier(_))(breakOut) : js.Array[VDomModifier])
-
+    CompositeModifier(js.Array(value.map(VDomModifier(_)).toIndexedSeq : _*))
```

Comment:
looks expensive to me. `map` produces a new `Array`, then this array is passed as varargs into `js.Array()` which afaik produces a defensive copy in 2.13, which is then copied again into a `js.Array`...

Reply:
We need to find some intermediate ground though, as breakout is gone in 2.13

https://www.scala-lang.org/blog/2017/02/28/collections-rework.html#views

We could do a bifurcated source tree, I don't have a lot of time for this right now though.

Reply:
I agree with both of you, some potentially expensive copying was introduced with regards to collections. But we also need to find some ground for supporting 2.13.

What is a "bifurcated source tree"?

Reply:
Just that it is possible to have for example a `src/main/scala_2.12` directory that provides alternate sources if compiling for scala 2.12.

Reply:
Ah, thanks! Now i know what you mean. I hope, we can avoid splitting the source tree, but let's see. I think, we just need to got through all the instances here where a copy might be introduced and check whether we can do something about it.